automatic/FirefoxESR/FirefoxESR.nuspec

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <version>0.0</version>
+    <authors>Mozilla</authors>
+    <projectUrl>https://www.mozilla.org/en-US/firefox/organizations/</projectUrl>
+    <projectSourceUrl>https://dxr.mozilla.org/comm-central/source/</projectSourceUrl>
+    <docsUrl>https://developer.mozilla.org/en-US/Firefox</docsUrl>
+    <bugTrackerUrl>https://bugzilla.mozilla.org/</bugTrackerUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/firefox.png</iconUrl>
+    <licenseUrl>https://www.mozilla.org/en-US/MPL/2.0/</licenseUrl>
+    <id>FirefoxESR</id>
+    <title>Mozilla Firefox ESR</title>
+    <owners>chocolatey</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+Bringing together all kinds of awesomeness to make browsing better for you.
+
+This package installs Firefox ESR in the first language which matches this list:
+
+1. Install arguments override parameter if present, e.g. `choco install Firefox -packageParameters "l=en-GB"`.
+  To get a list of all available locales have a look at this file: https://releases.mozilla.org/pub/firefox/releases/latest-esr/README.txt.
+2. If Firefox is already installed: the same language as the already installed Firefox.
+3. The Windows system language where the Firefox package gets installed.
+4. If Firefox does not support the system language, it will fall back to “en-US”
+
+Looking for Firefox Developer Edition? Install the [firefox-dev](/packages/firefox-dev) package.
+Looking for Firefox for Personal Use? Install the [Firefox](/packages/Firefox) package.
+    </description>
+    <summary>Bringing together all kinds of awesomeness to make browsing better for you</summary>
+    <tags>browser mozilla firefox admin foss cross-platform</tags>
+    <releaseNotes>https://www.mozilla.org/en-US/firefox/45.7.0/releasenotes/</releaseNotes>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/FirefoxESR</packageSourceUrl>
```

Comment:
Add projectSourceUrl (I think the source is in https://hg.mozilla.org/mozilla-central/)